### ISSUE TITLE
How to train different part of the model?

### ISSUE THREAD

**Issue by Ujang24** [matterport/Mask_RCNN#1248]
As we know that the training of this Mask R-CNN implementation is carried out to the head part of the model. How to train the other part(s) of the model?

Thanks

**Comment by waynezhang72** [comment]
Please refer to coco sample code to train the  other part(s) of the model as below:
    print("Fine tune Resnet stage 4 and up")
    model.train(dataset_train, dataset_val,
                learning_rate=config.LEARNING_RATE,
                epochs=90,
                layers='4+',
                augmentation=None)

    # Training - Stage 3
    # Fine tune all layers
    print("Fine tune all layers")
    model.train(dataset_train, dataset_val,
                learning_rate=config.LEARNING_RATE / 10,
                epochs=120,
                layers='all',
                augmentation=None)

**Comment by Ujang24** [comment]
Thank you @waynezhang72
